src/aws_credentials_file.erl

```diff
@@ -31,11 +30,12 @@ fetch(Options) ->
             parse_file(Path, Options)
     end.
 
--spec does_credentials_file_exist([option()]) -> {error, any()} | string().
+-spec does_credentials_file_exist(aws_credentials_provider:options()) ->
+        {error, any()} | string().
 does_credentials_file_exist(Options) ->
-    case maybe_add_home(proplists:get_value(<<"credential_path">>, Options, "/.aws/credentials")) of
+    case maybe_add_home(maps:get(credential_path, Options, "/.aws/credentials")) of
```

Comment:
The file backend has always been expecting a proplist. Replaced to use the new map format/

Reply:
:+1: